### DISCUSSION TITLE
Welcome to Member Handbook Discussions!

### DISCUSSION THREAD

**Discussion by andrewbolster** [FarsetLabs/member-handbook#9] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by chris18890** [comment]
Hi all, 

I'm Chris, been around since the beginning when we used to meet in The Parlour. [My Hackerspaces.org profile is here](https://wiki.hackerspaces.org/User:Chris18890). Most of what I work on revovles around Systems Administration, day-to-day operational stuff & automation/standardisation of these processes.
